messaging/pom.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus.ts.openshift</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>..</relativePath>
+    </parent>
+
+    <artifactId>messaging</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Quarkus OpenShift TS: Parent: Messaging</name>
+
+    <modules>
+        <module>artemis</module>
+        <module>artemis-jta</module>
+        <module>amqp-reactive</module>
+        <module>qpid</module>
+    </modules>
+</project>
```

Comment:
This parent seems totally useless? Big advantage of previous solution is that _all_ modules are listed on one place.

Reply:
Also, you can still list the modules by running "mvn validate".
From my point of view, having the common modules within a single parent is useful when you want to run a subset only. Moreover, we can delegate common configuration to these subset of modules by having the configuration in its parent.

Reply:
I prefer having modules defined in on place atm. `messaging/pom.xml` doesn't bring big additional value.

We can introduce it once we have more urgent need for such messaging parent. Similar situation is with https://github.com/quarkus-qe/quarkus-openshift-test-suite/tree/master/sql-db, https://github.com/quarkus-qe/quarkus-openshift-test-suite/tree/master/security etc.

Reply:
Ok, as you both think the same, I've reverted this change.